src/GameLogic/TurnRunner.ts

```diff
@@ -29,6 +30,17 @@ export class TurnRunner {
 		level.entities.getEntitiesOfType(EntityType.Fireball).forEach(entity => entity.update(turnState));
 		level.entities.getEntitiesOfType(EntityType.Iceblock).forEach(entity => entity.update(turnState));
 
+		for (let i = 0; i < level.width; i++) {
```

Comment:
I have conflicting feelings about this. I know I don't want to keep it like that in the long run, but I am now questioning whether we should make the exit an entity rather than a floor tile. Thinking back to how a similar issue is solved in DROD - the spikes. They're tracked in an array so that when the game wants to do the spike attack it just has to iterate through that one array rather than the whole level.
And I think that's how we should approach it.
Can you add TODO here referencing #69?